snapcraft/tests/plugins/python/test_python_finder.py

```diff
@@ -0,0 +1,114 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import re
+
+from testtools.matchers import (
+    Equals,
+    MatchesRegex,
+)
+
+from snapcraft.plugins._python import _python_finder
+
+from snapcraft import (
+    tests
+)
+
+
+class GetPythonCommandTestCase(tests.TestCase):
+
+    def test_staged(self):
+        """get_python_command should support staged python"""
+
+        stage_dir = 'stage_dir'
+        install_dir = 'install_dir'
+
+        # Create staged binary
+        _create_python_binary(stage_dir)
+
+        python_command = _python_finder.get_python_command(
+            'python2', stage_dir, install_dir)
+        self.assertThat(
+            python_command, Equals(os.path.join(
+                stage_dir, 'usr', 'bin', 'python2')))
+
+    def test_installed(self):
```

Comment:
Here you are using the word `installed`. I personally prefer that one over unstaged. Why not use it everywhere?

Reply:
Haha, you'll love this explanation: `installed` means "in the part." `unstaged` means actually installed on the host. Partway through this refactor I switched to using that language, I'll make that change here as well.

Reply:
Ha, and it's not even consistent here either. How dreadful.

Reply:
Fixed.